pkg/debuginfo/store.go

```diff
@@ -71,7 +73,7 @@ type Store struct {
 }
 
 // NewStore returns a new debug info store
-func NewStore(logger log.Logger, symbolizer *symbol.Symbolizer, config *Config) (*Store, error) {
+func NewStore(logger log.Logger, symbolizer *symbol.Symbolizer, config *Config, debuginfodClientCache *ObjectStorageDebugInfodClientCache) (*Store, error) {
```

Comment:
```suggestion
func NewStore(logger log.Logger, symbolizer *symbol.Symbolizer, config *Config, debuginfodClientCache DebugInfodClient) (*Store, error) {
```

I think it's better to use the interface here.